pkg/storaged/pages.jsx

```diff
@@ -536,54 +538,21 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                 cockpit.location.go(page.location);
         }
 
-        function is_clickable(element) {
-            return element.classList.contains("pf-m-clickable");
-        }
-
-        function next_clickable_sibling(element) {
-            do {
-                const next = element.nextElementSibling;
-                if (next && is_clickable(next))
-                    return next;
-                element = next;
-            } while (element);
-
-            return null;
-        }
+        function location_link(loc, children) {
+            if (!loc)
+                return children;
 
-        function previous_clickable_sibling(element) {
-            do {
-                const prev = element.previousElementSibling;
-                if (prev && is_clickable(prev))
-                    return prev;
-                element = prev;
-            } while (element);
+            // HACK - somehow the link styling gets lost when putting
+            // things in a Truncate. Put it back.
+            const decorated_children = (
```

Comment:
We should do this in CSS and not through style. But either way a note is good
```suggestion
            // HACK - somehow the link styling gets lost when putting
            // things in a Truncate. Put it back.
            // https://github.com/patternfly/patternfly/issues/7456
            const decorated_children = (
```

Reply:
Yay, thanks for filing the bug! I have kinda given up on PF bug filing because of their autoclose shenanigans, but it's of course the right thing to do.

Reply:
I put this into its own commit and into storage.scss.